lib/elixir/test/elixir/map_test.exs

```diff
@@ -13,6 +13,14 @@ defmodule MapTest do
 
   @map %{a: 1, b: 2}
 
+  test "new/1" do
+    assert Map.new([a: 1, b: 2]) == @map
```

Comment:
I think we should assert explicitly with `%{a: 1, b: 2}` here, `@map` is only for `"maps in attributes"` test.